### ISSUE TITLE
Failed at the bufferutil@1.2.1 install script 'node-gyp rebuild'

### ISSUE THREAD

**Issue by ramneekh** [notorii/generator-mean-seed#31]
I am stuck with is error...
33071 error Windows_NT 6.1.7601
33072 error argv "node" "C:\\Users\\rh\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "yo" "bower" "grunt-cli" "jasmine-node" "karma" "yuidocjs" "forever" "less"
33073 error node v0.12.0
33074 error npm  v3.3.5
33075 error code ELIFECYCLE
33076 error bufferutil@1.2.1 install: `node-gyp rebuild`
33076 error Exit status 1
33077 error Failed at the bufferutil@1.2.1 install script 'node-gyp rebuild'.
33077 error This is most likely a problem with the bufferutil package,
33077 error not with npm itself.
33077 error Tell the author that this fails on your system:
33077 error     node-gyp rebuild
33077 error You can get their info via:
33077 error     npm owner ls bufferutil
33077 error There is likely additional logging output above.
33078 verbose exit [ 1, true ]

Please advice...

**Comment by lukemadera** [comment]
Yeah I think I had this issue with Windows for a bit - you'll have to do some googling and installing of Windows specific tools to support this, i.e.:
https://www.robertkehoe.com/2015/03/fix-node-gyp-rebuild-error-on-windows/

**Comment by macliems** [comment]
Install Visual Studio 2015 Community, "create" C++ project, get web tools. This solved my problem with installing last version of mongodb.
